### ISSUE TITLE
Submitting an app for review via deliver fails because iTunes Connect expects a specific language code.

### ISSUE THREAD

**Issue by craigmarvelley** [fastlane/fastlane#6226]
### New Issue Checklist
- [x] Updated fastlane to the latest version
- [x] I have read the [Contribution Guidelines](https://github.com/fastlane/fastlane/blob/master/CONTRIBUTING.md)
### Issue Description

When submitting a build to iTunes Connect for app review, I'm getting the following error:

`You must detail what's new in this version in English (U.K.).`

I have followed the instructions at https://github.com/fastlane/fastlane/blob/master/deliver/README.md and have my metadata in the following folder structure:

default
- keywords.txt
- marketing_url.txt
- name.txt
- privacy_url.txt
- support_url.txt
- release_notes.txt

Shouldn't the release notes file I have therefore apply to all language codes?
##### Complete output when running fastlane, including the stack trace and command used

Trimmed to the important bit:

```
[06:57:25]: [32m---------------------[0m
[06:57:25]: [32m--- Step: deliver ---[0m
[06:57:25]: [32m---------------------[0m
[06:57:25]: Login to iTunes Connect (<redacted>)
[06:57:27]: Login successful

+--------------------------------------------------------------------+-----------------------------+
|                                      [32mdeliver 1.13.3 Summary[0m                                      |
+--------------------------------------------------------------------+-----------------------------+
| submit_for_review                                                  | true                        |
| automatic_release                                                  | false                       |
| skip_screenshots                                                   | true                        |
| force                                                              | true                        |
| app_version                                                        | 1.11.2                      |
| submission_information.add_id_info_uses_idfa                       | false                       |
| submission_information.content_rights_contains_third_party_content | false                       |
| submission_information.content_rights_has_rights                   | false                       |
| ipa                                                                | /Users/jenkins/workspace... |
| app_identifier                                                     | <redacted> |
| screenshots_path                                                   | ./fastlane/screenshots      |
| metadata_path                                                      | ./fastlane/metadata         |
| username                                                           | <redacted> |
| skip_binary_upload                                                 | false                       |
| skip_metadata                                                      | false                       |
| dev_portal_team_id                                                 | <redacted>                  |
+--------------------------------------------------------------------+-----------------------------+

[06:57:27]: Making sure the latest version on iTunes Connect matches '1.11.2' from the ipa file...
[06:57:31]: [32mSuccessfully set the version to '1.11.2'[0m
[06:57:31]: Loading './fastlane/metadata/default/description.txt'...
[06:57:31]: Loading './fastlane/metadata/default/keywords.txt'...
[06:57:31]: Loading './fastlane/metadata/default/release_notes.txt'...
[06:57:31]: Loading './fastlane/metadata/default/support_url.txt'...
[06:57:31]: Loading './fastlane/metadata/default/marketing_url.txt'...
[06:57:31]: Loading './fastlane/metadata/default/name.txt'...
[06:57:31]: Loading './fastlane/metadata/default/privacy_url.txt'...
[06:57:31]: Detected languages: ["default"]
[06:57:33]: Uploading metadata to iTunes Connect
[06:57:34]: [32mSuccessfully uploaded initial set of metadata to iTunes Connect[0m
[06:57:36]: Uploading binary to iTunes Connect
[06:57:36]: Going to upload updated app to iTunes Connect
[06:57:36]: [32mThis might take a few minutes. Please don't interrupt the script.[0m
[06:59:35]: [32miTunes Transporter successfully finished its job[0m
[06:59:35]: [32m------------------------------------------------------------------------------------------------------[0m
[06:59:35]: [32mSuccessfully uploaded package to iTunes Connect. It might take a few minutes until it's visible online.[0m
[06:59:35]: [32m------------------------------------------------------------------------------------------------------[0m
[06:59:35]: [32mFinished the upload to iTunes Connect[0m
[06:59:37]: Selecting the latest build...
[06:59:38]: Waiting iTunes Connect processing for build 1.11.2 (2571.19)... this might take a while...
[07:00:09]: Waiting iTunes Connect processing for build 1.11.2 (2571.19)... this might take a while...
[07:00:41]: Waiting iTunes Connect processing for build 1.11.2 (2571.19)... this might take a while...
[07:01:13]: Waiting iTunes Connect processing for build 1.11.2 (2571.19)... this might take a while...
[07:01:45]: Waiting iTunes Connect processing for build 1.11.2 (2571.19)... this might take a while...
[07:02:17]: Waiting iTunes Connect processing for build 1.11.2 (2571.19)... this might take a while...
[07:02:49]: Waiting iTunes Connect processing for build 1.11.2 (2571.19)... this might take a while...
[07:03:21]: Waiting iTunes Connect processing for build 1.11.2 (2571.19)... this might take a while...
[07:03:52]: Waiting iTunes Connect processing for build 1.11.2 (2571.19)... this might take a while...
[07:04:24]: Waiting iTunes Connect processing for build 1.11.2 (2571.19)... this might take a while...
[07:04:56]: Waiting iTunes Connect processing for build 1.11.2 (2571.19)... this might take a while...
[07:04:56]: 
[07:04:56]: You can tweet: "iTunes Connect #iosprocessingtime 5 minutes"
[07:05:29]: Waiting iTunes Connect processing for build 1.11.2 (2571.19)... this might take a while...
[07:05:29]: 
[07:05:29]: You can tweet: "iTunes Connect #iosprocessingtime 6 minutes"
[07:06:01]: Selecting build 1.11.2 (2571.19)...
[07:06:02]: [32mSuccessfully selected build[0m
[07:06:02]: Submitting the app for review...
[07:06:05]: [33m[33mVariable Dump:[0m[0m
[07:06:05]: {:ENVIRONMENT=>"prod", :PLATFORM_NAME=>nil, :LANE_NAME=>"deploy_production_build", :SIGH_PROFILE_TYPE=>"app-store", :BUILD_NUMBER_REPOSITORY=>"<redacted>", :IPA_OUTPUT_PATH=>"/Users/jenkins/workspace/workspace/ios-release/build/<redacted>“, :DSYM_OUTPUT_PATH=>"/Users/jenkins/workspace/workspace/ios-release/build/<redacted>.app.dSYM.zip", :XCODEBUILD_ARCHIVE=>"/Users/jenkins/Library/Developer/Xcode/Archives/2016-09-21/<redacted> 2016-09-21 06.55.33.xcarchive"}
[07:06:05]: [31mYou must detail what's new in this version in English (U.K.).[0m

+------+-------------------------------------+-------------+
|                     [32mfastlane summary[0m                     |
+------+-------------------------------------+-------------+
| Step | Action                              | Time (in s) |
+------+-------------------------------------+-------------+
| 1    | Verifying required fastlane version | 0           |
| 2    | ensure_git_branch                   | 0           |
| 3    | match                               | 1           |
| 4    | get_build_number_repository         | 0           |
| 5    | gym                                 | 114         |
| 6    | [31mdeliver[0m                             | 519         |
+------+-------------------------------------+-------------+

[07:06:05]: [31mfastlane finished with errors[0m

Build step 'Execute shell' marked build as failure
[ssh-agent] Stopped.
Finished: FAILURE
```
### Configuration Files

Please copy the complete content of your `Fastfile` and any other configuration files you use below:

Fastfile:

``` ruby
desc "Submits a Production build to the App Store"
lane :deploy_production_build do |options|

  ensure_git_branch

  match(
    readonly: true,
    type: 'appstore' 
  )

  gym(
    output_name: "<redacted>.ipa",
    include_symbols: true,
    include_bitcode: false
  )

  deliver(
    submit_for_review: true,
    automatic_release: false,
    skip_screenshots: true,
    force: true,
    app_version: options[:app_version],
    submission_information: {
      add_id_info_uses_idfa: false,
      content_rights_contains_third_party_content: false,
      content_rights_has_rights: false
    }
  )

  add_git_tag(
    tag: options[:app_version]
  )

  push_git_tags

  clean_build_artifacts

end
```

**Comment by TKBurner** [comment]
@craigmarvelley Great to hear from you. Can you test a few things for me. What happens if you have an english folder in addition to the default folder. Does it succeed? I want to see if this only holds for english. Thanks!

**Comment by craigmarvelley** [comment]
Hey, sorry it took a while to reply – had to wait until I needed to submit another build! Yep, if I have an en-GB folder in addition to the default folder it works fine.

**Comment by TKBurner** [comment]
@craigmarvelley Thanks for following up. Are you able to reproduce this on other projects? I've reached out to the core team to get them to look into this. Thanks!

**Comment by craigmarvelley** [comment]
Close this if you want, but as far as I know it's not fixed. You're not waiting on anything from me are you? I can reproduce this on other projects yep. As far as I was concerned it was with the core team.
